### ISSUE TITLE
Strategy for namespace migration

### ISSUE THREAD

**Issue by bpaczkowski** [wojtpl2/ExtendedXmlSerializer#226]
Has anyone tried doing that? I have one migration behind me, and the way I did it was basically copy an old version of the library that contains the serialized objects, deserialize the XMLs, map them to the new version and serialize them back, but writing such a migrator is cumbersome and I have another migration ahead of me. Is there a better solution in ExtendedXmlSerializer for this?

**Comment by Mike-EEE** [comment]
Ah that's a good one, @bpaczkowski ... I myself am not too familiar with the migration story, as it is more of a v1 feature.  We did port it to v2, but mostly for legacy reasons.  I believe the feeling was that we would flesh it out proper for v2 at some point, but you know how that goes. 😅

To make sure I understand the question, you're wanting to migrate an entire namespace, correct?

**Comment by bpaczkowski** [comment]
Yes that is correct. The easiest way would be search and replace on the XML string itself, but that only works if a prefix on a namespace changes and the structure in the namespace itself doesn't, which isn't always the case.
